src/Data/CurrentUser/CurrentUserDataItem.ts

```diff
@@ -184,8 +190,8 @@ async function getTokenBasedCurrentUser(tokenAuthorization: string) {
   if (!response.ok) {
     const errorMessage =
       response.status === 401
-        ? 'The link you followed is invalid or has expired. Please request a new one.'
-        : 'Failed to fetch user profile.'
+        ? localizeExpiredMessage(lang)
+        : localizeFailedVerify(lang)
```

Comment:
* [ ] FYI the logic around the error messages is duplicated and could become part of a single shared helper instead of two.